ggircs-app/app/components/FuelManagement/CreateEditFuelChargeRow.tsx

```diff
@@ -0,0 +1,261 @@
+    clearStateData();
+    if (operation === "edit") {
+      setIsEditing(false);
+      updateFuelCharge();
+    } else {
+      addFuelCharge();
+    }
+  };
+
+  return (
+    <>
+      <tr>
+        <td colSpan={5}>
+          {operation === "edit" ? (
+            <Alert
+              id={`edit-row-aria-label-${charge?.id}`}
+              aria-label={`edit-row-label-${charge?.id}`}
+              variant="warning"
+            >
+              Editing Row
+            </Alert>
+          ) : (
+            <Alert variant="info">Add a New Rate Period</Alert>
+          )}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <DatePicker
+            label="Period Start"
+            name="start-date"
+            size="small"
+            value={startPeriodData}
+            onChange={(e: ChangeEvent<HTMLInputElement>) =>
+              setStartPeriodData(e.target.value)
+            }
+            onBlur={(e: ChangeEvent<HTMLInputElement>) =>
+              handleValidate("startPeriod", e.target.value)
+            }
+          />
+          {startPeriodHasError && (
+            <p style={{ fontSize: "0.8rem", color: "#cd2026" }}>
+              Date overlaps with an existing period
+            </p>
+          )}
+        </td>
+        <td>
+          <DatePicker
+            label="Period End"
+            name="end-date"
+            size="small"
+            value={endPeriodData}
+            onChange={(e: ChangeEvent<HTMLInputElement>) =>
+              setEndPeriodData(e.target.value)
+            }
+            onBlur={(e: ChangeEvent<HTMLInputElement>) =>
+              handleValidate("endPeriod", e.target.value)
+            }
+          />
+          {endPeriodHasError && (
+            <p style={{ fontSize: "0.8rem", color: "#cd2026" }}>
+              Date overlaps with an existing period
+            </p>
+          )}
+        </td>
+        <td>
+          <Input
+            label="Charge"
+            size="small"
+            name="charge"
+            type="number"
+            value={fuelChargeData}
+            onChange={(e: ChangeEvent<HTMLInputElement>) =>
+              setFuelChargeData(e.target.value)
+            }
+          />
+        </td>
+        <td className="icon-cell" colSpan={2}>
+          <Row>
+            <Col>
+              <OverlayTrigger
+                placement="top"
+                delay={{ show: 250, hide: 400 }}
+                overlay={<Tooltip id="save-tooltip">Save</Tooltip>}
+              >
+                <div>
+                  <FontAwesomeIcon
+                    role="button"
+                    name="save"
+                    className={
+                      disableSaveButton
+                        ? "save-button-disabled"
+                        : "save-cancel-button"
+                    }
+                    size="lg"
+                    icon={faCheck}
+                    onClick={handleSave}
+                  />
```

Comment:
Should we use the BCGov button style here?

Reply:
I've colored the icons bcgov blue. Do you mean use a button instead of an icon here?

Reply:
Yes, I noticed the icons don't stand out as "clickable". maybe @suhafa can advise if it's worth it or if it's ok for now, and we can address that in a later PR?

Reply:
Cool, I'll make some of the other small changes listed in the PR. My thought behind the use of icons here is I was trying to maximize the use of a small space. I originally had buttons but they looked clunky in the small areas where the icons are currently.

Reply:
@suhafa here are what the buttons look like
<img width="1009" alt="image" src="https://user-images.githubusercontent.com/11507754/165582861-0d2f4c78-132c-4b94-8ed9-174eeac9c1b6.png">